src/terrain/node_group.cpp

```diff
@@ -1,84 +1,129 @@
-#include <cstdint>
 #include "node_group.hpp"
 
+#include <cstdint>
+
 namespace terrain {
 
-NodeGroup::NodeGroup(Tile* tile, UnitPath path_type){
+NodeGroup::NodeGroup(Tile* tile, UnitPath path_type)
+{
     tiles.insert(tile);
     center_x = tile->get_x();
     center_y = tile->get_y();
     center_z = tile->get_z();
     path_type_ = path_type;
 }
 
-void NodeGroup::add_adjacent(NodeGroup* NG, UnitPath type){
-    if (NG != this){
+void
+NodeGroup::add_adjacent(NodeGroup* NG, UnitPath type)
+{
+    if (NG != this) {
         adjacent.insert(std::make_pair(NG, type));
         NG->adjacent.insert(std::make_pair(this, type));
     }
 }
-void NodeGroup::remove_adjacent(NodeGroup* NG){
-    if (NG != this){
+
+void
+NodeGroup::remove_adjacent(NodeGroup* NG)
+{
+    if (NG != this) {
         adjacent.erase(NG);
         NG->adjacent.erase(this);
     }
 }
 
-std::map<const NodeGroup *, UnitPath> NodeGroup::get_adjacent_map() const{
-    return std::map<const NodeGroup *, UnitPath>(adjacent.begin(), adjacent.end());
+std::map<const NodeGroup*, UnitPath>
+NodeGroup::get_adjacent_map() const
+{
+    return std::map<const NodeGroup*, UnitPath>(adjacent.begin(), adjacent.end());
 }
 
-std::map<NodeGroup *, UnitPath> NodeGroup::get_adjacent_map(){
+std::map<NodeGroup*, UnitPath>
+NodeGroup::get_adjacent_map()
+{
     return adjacent;
 }
 
-std::set<const NodeGroup *> NodeGroup::get_adjacent_clear(int path_type)const{
-    std::set<const NodeGroup *> out;
+std::set<const NodeGroup*>
+NodeGroup::get_adjacent_clear(int path_type) const
+{
+    std::set<const NodeGroup*> out;
 
-    for (const std::pair<NodeGroup *,UnitPath> t : adjacent) {
+    for (const std::pair<NodeGroup*, UnitPath> t : adjacent) {
         if (t.second.compatible(path_type) && t.second.is_open()) {
             out.insert(t.first);
         }
     }
     return out;
 }
 
-std::map<NodeGroup *, UnitPath> NodeGroup::merge_groups(NodeGroup other){
-    center_x = (center_x*tiles.size() + other.center_x*other.tiles.size())/(tiles.size()+other.tiles.size());
-    center_y = (center_y*tiles.size() + other.center_y*other.tiles.size())/(tiles.size()+other.tiles.size());
-    center_z = (center_z*tiles.size() + other.center_z*other.tiles.size())/(tiles.size()+other.tiles.size());
+std::map<NodeGroup*, UnitPath>
+NodeGroup::merge_groups(NodeGroup other)
+{
+    center_x = (center_x * tiles.size() + other.center_x * other.tiles.size())
+               / (tiles.size() + other.tiles.size());
+    center_y = (center_y * tiles.size() + other.center_y * other.tiles.size())
+               / (tiles.size() + other.tiles.size());
+    center_z = (center_z * tiles.size() + other.center_z * other.tiles.size())
+               / (tiles.size() + other.tiles.size());
```

Comment:
If you really care about the length of this line, change center_a to ca or something.